### PR TITLE
docs(readme): add plugin compatibility table

### PR DESCRIPTION
**Pull request by Fdawgs**
Inspired by what @Eomm does with his personal Fastify plugins, this should make it clearer for users.
Also fixed a tiny grammatical error.

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Fdawgs** [comment]
Could also add a note stating fixes/features are not backported?

File: README.md [hunk 1]
```diff
@@ -7,19 +7,22 @@
...
 ```
 
+### Compatibility
+
+| Plugin version | Fastify version |
+| -------------- |---------------- |
+| `^1.0.0`       | `^1.0.0`        |
+| `^3.0.0`       | `^2.0.0`        |
+| `^7.0.0`       | `^3.0.0`        |
+| `^8.0.0`       | `^4.0.0`        |
```
Comment (Eomm) [inline README.md:22]: I was thinking that the reverse order is better for the user (what the user wants, goes first)

```suggestion
| `^8.0.0`       | `^4.0.0`        |
| `^7.0.0`       | `^3.0.0`        |
| `^3.0.0`       | `^2.0.0`        |
| `^1.0.0`       | `^1.0.0`        |
```

**Review by Eomm — COMMENTED** [review]
> Could also add a note stating fixes/features are not backported?

I would write a sentence with a link to the fastify LTS document.
So, if a fastify version is out of the support, there are not backport actions

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**